plugin-server/src/main/services/http-server.ts

```diff
@@ -50,6 +44,24 @@ export function createHttpServer(hub: Hub, serverInstance: ServerInstance, serve
                 res.statusCode = 503
                 res.end(JSON.stringify(responseBody))
             }
+        } else if (req.url === '/_ready' && req.method === 'GET') {
```

Comment:
What's the difference between `_health` and `_ready` and when do we use one or the other? Typically we have a livenessProbe which is just the verify that the container is completely dead and a readinessProbe which will for instance let us know when we can consider decommissioning the other pods in a deployment.

Maybe will be clearer with the Helm Chart changes also.

Reply:
`_health` = livenessProbe, e2e check that kafka works
`_ready` = readinessProbe, checks that the Kafka consumer initially joined the group

Reply:
Ok. Liveness is going to trigger a restart of the container, which we normally do if we know the container is dead and will not recover. For instance, the process is completely deadlocked.

I'm not really sure what the best thing to do for a Kafka consumer is however. The risk is we end up killing containers as they are in the process of rebalancing etc i.e. actively recovering but I don't know enough to be concrete.